internal/internal_workflow_testsuite.go

```diff
@@ -1111,16 +1111,8 @@ func (env *testWorkflowEnvironmentImpl) RequestCancelLocalActivity(activityID st
 		env.logger.Debug("RequestCancelLocalActivity failed, LocalActivity not exists or already completed.", zap.String(tagActivityID, activityID))
 		return
 	}
-	activityInfo := env.getActivityInfo(activityID, getActivityFunctionName(env.registry, task.params.ActivityFn))
 	env.logger.Debug("RequestCancelLocalActivity", zap.String(tagActivityID, activityID))
-	delete(env.localActivities, activityID)
-	env.postCallback(func() {
-		lar := &localActivityResultWrapper{err: ErrCanceled, backoff: noRetryBackoff}
-		task.callback(lar)
-		if env.onLocalActivityCanceledListener != nil {
```

Comment:
What was this?

Reply:
For some reason the cancellation path in the testEnv was different for local activity and the real one. I unified them. I think it is because the local activity was added later.